src/ua/hillel/Cat.java

```diff
@@ -1,4 +1,4 @@
-public class Cat extends Animal {
+public class  Cat extends Animal {
     private static int maxRun = 200;
     private static int maxSwim = 0;
     private static int CatCount;
```

Comment:
Class' members should have a specific order:

constants
static variables
instance variables
constructors
getters + setters
public methods
private methods
Please change this class' members order according to the info in this comment.